assets/js/util/when-in-view-context.js

```diff
@@ -0,0 +1,84 @@
+/**
```

Comment:
I know we have HOCs with this filename convention, but we _should_ rename them; let's not keep doing it 😅 

We also have `assets/js/util/whenScopesGranted.js`, so either way we have to settle on a consistent naming, but let's use the more "obvious"/"consistent" format instead of the dashes when possible.

Reply:
Yeah, makes sense. Renamed